### PR TITLE
feat : added hairStyle "turban"

### PR DESCRIPTION
**Pull request by itskaustubh**
Added turban as a hair style from the Avatar Illustration System, as some of my friends wear one :)

### CODE REVIEW

File: src/hair/turban.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+export default function (props) {
+  const { color } = props;
+  return (
+    <svg 
+      style={{
+		position: "absolute",
+		bottom: "34%",
```
Comment (wjq990112) [inline src/hair/turban.js:9]: Should set width and height in style, the position of turban is incorrect.

**Comment by wjq990112** [comment]
Have you test after edit? Now is incorrect.

**Comment by wwayne** [comment]
Thanks @itskaustubh @wjq990112 :)
Merging, I will find time to do the fix for you before the next version released
I was thinking to add a new option named hairStyle

**Comment by itskaustubh** [comment]
Thanks @wwayne for taking the time out and fixing it
Thanks @wjq990112 for pointing out the mistake.
:)

**Pull request merged**